### PR TITLE
Fix typo and history error

### PR DESCRIPTION
**Pull request by bkmgit**
- use up instead of left arrow key to get previous command
- use history that is similar to what one would obtain based on following these instructions

### CODE REVIEW

File: _episodes/05-loop.md [hunk 1]
```diff
@@ -630,7 +630,7 @@ $ for datafile in NENE*A.txt NENE*B.txt; do echo $datafile stats-$datafile; done
 ~~~
 {: .language-bash}
 
-Using the left arrow key,
+Using the up arrow key,
```
Comment (gcapes) [inline _episodes/05-loop.md:633]: She's already pressed the up arrow key. At this point, she's making edits, using the left arrow key.

**Review by gcapes — CHANGES_REQUESTED** [review]
History edits look fine, but the arrow key presses were already correct.

File: _episodes/05-loop.md (hunk 1 above)
Comment (bkmgit) [reply _episodes/05-loop.md]: Thanks. Updated it, tried to clarify.

**Review by gcapes — CHANGES_REQUESTED** [review]
Perhaps this would read more easily as
> Nelle navigates to the `echo` command and changes it to `bash goostats.sh`

**Pull request merged**